cdk/src/stack.ts

```diff
@@ -11,6 +11,7 @@ import { getSvelteKitEnvironmentVariables } from './support'
 import { AmplifyAwsSubdomain } from './components/amplify-aws-subdomain'
 import { SupportBox } from './components/support-box'
 import type { AmplifyAwsSubdomainProps } from './components/amplify-aws-subdomain'
+import { PrivacyNotificationFunction } from './components/privacy-notification-function/privacy-notification-function'
```

Comment:
```suggestion
import { PrivacyNotificationFunction } from './components/privacy-notification-function/privacy-notification-function'
import type { AmplifyAwsSubdomainProps } from './components/amplify-aws-subdomain'
```

`type` imports should be last in the import list (eslint rule TBD 😬 )